docs/index.md

```diff
@@ -258,3 +258,15 @@ To get started, see the [Provider Example Configs](https://github.com/couchbasec
 
   Manage Storage Auto-expansion for Azure Clusters.
 
+* [Create Audit Log Export Job in Capella](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/main/examples/audit_log_export)
+
+  Create Audit Log Export Jobs.
+
+* [Manage Audit Log Settings in Capella](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/main/examples/audit_log_settings)
+   
+  Manage Audit Log Settings.
+
+* [List Audit Logs EventIds in Capella](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/main/examples/audit_logs_event_ids)
```

Comment:
Not a blocker for this release, but note that this list of Example Usage links is growing too long for easy parsing, and in the near future it would be good to group them in related sections.